### PR TITLE
[Bug][Core] Fix Change SeaTunnel Logo Print Error in some jdk version

### PR DESCRIPTION
**Pull request by john8628**
## Purpose of this pull request

   https://github.com/apache/incubator-seatunnel/issues/3153
## Check list

* [x] Code changed are covered with tests, or it does not need tests for reason:
* [x] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/incubator-seatunnel/tree/dev/docs

### CODE REVIEW

File: seatunnel-core/seatunnel-core-base/src/main/java/org/apache/seatunnel/core/base/command/BaseTaskExecuteCommand.java [hunk 1]
```diff
@@ -96,7 +95,8 @@ protected final void close(List<? extends Plugin<E>>... plugins) {
     protected void showAsciiLogo() {
         String printAsciiLogo = System.getenv("SEATUNNEL_PRINT_ASCII_LOGO");
         if ("true".equalsIgnoreCase(printAsciiLogo)) {
-            AsciiArtUtils.printAsciiArt(Constants.LOGO);
```
Comment (hailin0) [inline seatunnel-core/seatunnel-core-base/src/main/java/org/apache/seatunnel/core/base/command/BaseTaskExecuteCommand.java:99]: Remove unused code?

https://github.com/apache/incubator-seatunnel/blob/dev/seatunnel-core/seatunnel-core-base/src/main/java/org/apache/seatunnel/core/base/utils/AsciiArtUtils.java
https://github.com/apache/incubator-seatunnel/blob/dev/seatunnel-core/seatunnel-core-starter/src/main/java/org/apache/seatunnel/core/starter/utils/AsciiArtUtils.java

**Comment by TyrantLucifer** [comment]
Please refer to [Coding Guide](https://seatunnel.apache.org/docs/contribution/coding-guide#how-to-submit-a-high-quality-pull-request) and change your pull request titile. Thx.

File: seatunnel-core/seatunnel-core-base/src/main/java/org/apache/seatunnel/core/base/command/BaseTaskExecuteCommand.java (hunk 1 above)
Comment (john8628) [reply seatunnel-core/seatunnel-core-base/src/main/java/org/apache/seatunnel/core/base/command/BaseTaskExecuteCommand.java:99]: ok ,that sounds good;

**Review by hailin0 — APPROVED** [review]
LGTM

**Review by TyrantLucifer — APPROVED** [review]
+1

**Review by EricJoy2048 — APPROVED** [review]
LGTM

**Pull request merged**